packages/sdk/src/transactions/Inscriber.ts

```diff
@@ -232,7 +259,7 @@ export class Inscriber extends PSBTBuilder {
     this.restrictUsageInPreviewMode()
     this.isBuilt()
 
-    const amount = this.fee + this.outputAmount
+    const amount = this.recovery ? this.outputAmount - this.fee : this.outputAmount + this.fee
```

Comment:
do we not need to add `postage` here like previous fn ? or is it already added into one of the amount

Reply:
postage is already added in the `outputs` which is then reduced and summed into `outputAmount`.